kuma/wiki/models.py

```diff
@@ -568,6 +568,10 @@ def build_json_data(self):
         if self.pk:
             for translation in self.other_translations:
                 revision = translation.current_revision
+                if revision.summary:
+                   summary = revision.summary
+                else:
+                   summary = translation.get_summary(strip_markup=False)
```

Comment:
oh, duh. I was just looking at the green lines so I ignored that 570 sets `revision` to `translation.current_revision`